ClassNotFoundException for Runtime

**Describe the bug**

Exception:

```
Caused by: java.lang.ClassNotFoundException: Didn't find class "oolong.Oolong$Runtime$1" on path: DexPathList...
```

Cause appears to be an issue somewhere in the compilation pipeline which incorrectly cases the class name. Here is the class name in the dex file:

```
Class #5399            -
  Class descriptor  : 'Loolong/Oolong$runtime$1;'
  Access flags      : 0x0010 (FINAL)
  Superclass        : 'Lkotlin/jvm/internal/Lambda;'
  Interfaces        -
    #0              : 'Lkotlin/jvm/functions/Function0;'
```

Notice `oolong.Oolong$Runtime$1` vs `oolong/Oolong$runtime$1`; `R` vs `r` in runtime.

--- a/oolong/src/commonMain/kotlin/oolong/Oolong.kt
+++ b/oolong/src/commonMain/kotlin/oolong/Oolong.kt
@@ -2,7 +2,6 @@ package oolong
 
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.SupervisorJob
 import kotlinx.coroutines.cancel
 import kotlinx.coroutines.isActive
@@ -23,11 +22,11 @@ object Oolong {
         view: View<Model, Props>,
         render: Render<Msg, Props>
     ): Dispose {
-        val runtime = Runtime(init, update, view, render)
+        val runtime = RuntimeImpl(init, update, view, render)
         return { runtime.dispose() }
     }
 
-    private class Runtime<Model : Any, Msg : Any, Props : Any>(
+    private class RuntimeImpl<Model : Any, Msg : Any, Props : Any>(
         init: Init<Model, Msg>,
         private val update: Update<Model, Msg>,
         private val view: View<Model, Props>,

